src/mongoose_async_pools.erl

```diff
@@ -44,6 +46,32 @@ config_spec() ->
 pool_name(HostType, PoolId) ->
     persistent_term:get({?MODULE, HostType, PoolId}).
 
+-spec sync(mongooseim:host_type(), pool_id()) -> term().
+sync(HostType, PoolId) ->
+    Pids = mongoose_async_pools:get_workers(HostType, PoolId),
```

Comment:
Doesn't need that fully-qualified name. And then so, maybe this function doesn't need exporting and also doesn't need the ignore_xref.